### PR TITLE
Add pipeline file on branch 202012, supporting PR build & official build

### PR DESCRIPTION
**Pull request by liushilongbuaa**
**- Why I did it**
Add pipeline file for Azure pipeline. PR build & official build
**- How I did it**

**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: pipelines/official_build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+stages:
+- stage: One
```
Comment (lguohan) [inline pipelines/official_build.yml:7]: what is stage One?

File: pipelines/official_build.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+stages:
```
Comment (lguohan) [inline pipelines/official_build.yml:6]: what is the trigger?

**Comment by lguohan** [comment]
i suggest to start form official build on master branch first, then we can start doing other branches.

File: pipelines/official_build.yml (hunk 2 above)
Comment (liushilongbuaa) [reply pipelines/official_build.yml:6]: trigger can be overide by UI config. And I have tried yaml format trigger: [schedules](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml), which not work as expected.
So I perfer [UI config.](https://dev.azure.com/sonicswitch/build/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=33&view=Tab_Triggers)

File: pipelines/official_build.yml (hunk 1 above)
Comment (liushilongbuaa) [reply pipelines/official_build.yml:7]: I worried that use different stages maybe run on different node. 
This worry comes from log, and haven't confirmed by ms doc.
So I prefer to use one stage. it is a non-sense name

Comment (liushilongbuaa) [reply pipelines/official_build.yml]: changed to Build

**Pull request closed without merging**